twitterscraper/query.py

```diff
@@ -135,7 +135,7 @@ class NoElement: pass
 def roundup(numerator, denominator):
     return numerator // denominator + (numerator % denominator > 0)
 
-def query_tweets(query, limit = None, begindate = dt.date(2017,1,1), enddate = dt.date.today(), poolsize = 20):
+def query_tweets(query, limit = None, begindate = dt.date(2017,1,1), enddate = dt.date.today(), poolsize = 20, lang = ''):
```

Comment:
you're not using spacing around `=` consistently, PEP8 suggests no spaces around `=` within param lists